app/src/main/java/com/codesoom/assignment/model/Product.java

```diff
@@ -0,0 +1,68 @@
+package com.codesoom.assignment.model;
+
+import com.google.common.base.Objects;
+
+public class Product {
+    private Long Id;
```

Comment:
```suggestion
    private Long id;
```
별거아니지만 변수의 변수 앞글자는 소문자로 만들어주세요!

참고자료: https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html